Makefile

```diff
@@ -26,6 +26,11 @@ test_yul: warp
 	python -m pytest tests/behaviour/ -v --tb=short --workers=auto $(ARGS)
 .PHONY: test_yul
 
+test_semantics: warp
+	tests/semantic/init_test.sh
+	python -m pytest tests/semantic/test_semantics.py -v --tb=short -n auto $(ARGS)
```

Comment:
`-n auto` why not `--workers auto`? It reuses our current dependency on `pytest-parallel` and doesn't add a dependency on `pytest-xdist`

Reply:
comment above addresses this

Reply:
Maybe then replace all usages of pytest-parallel with pytest-xdist?

Reply:
done